app/components/contact-modal/contact-modal.tsx

```diff
@@ -69,13 +67,13 @@ const ContactModal: React.FC<Props> = ({
       hidden: supportChannelsToHide?.includes(SupportChannels.StatusPage),
     },
     {
-      name: LL.support.telegram(),
-      icon: <TelegramOutline width={24} height={24} fill={colors.black} />,
+      name: LL.support.Discord(),
```

Comment:
LL.support.Discord() does not exist, so you can create it in the internationalization (i18n) file structure, and give it at least the English and Spanish translations. 

Wherever the LL.support.telegram was, is prob where you can add this.